src/Microsoft.ML.Data/Data/Conversion.cs

```diff
@@ -1826,8 +1821,8 @@ public void Convert(ref TX span, ref DZ value)
         public void Convert(ref BL src, ref I2 dst) => dst = (I2)src;
         public void Convert(ref BL src, ref I4 dst) => dst = (I4)src;
         public void Convert(ref BL src, ref I8 dst) => dst = (I8)src;
-        public void Convert(ref BL src, ref R4 dst) => dst = (R4)src;
-        public void Convert(ref BL src, ref R8 dst) => dst = (R8)src;
+        public void Convert(ref BL src, ref R4 dst) => dst = System.Convert.ToSingle(src);
```

Comment:
yep, it doesn't seem to recognize otherwise. 

---
In reply to: [211109280](https://github.com/dotnet/machinelearning/pull/692#discussion_r211109280) [](ancestors = 211109280)